### ISSUE TITLE
fabricjs4 _setObjectScale error when two finger on active object

### ISSUE THREAD

**Issue by ShahramCodeDeveloper** [fabricjs/fabric.js#6377]
## Version
Version 4.0.0-beta.12

## Test Case
http://jsfiddle.net/ShahramCodeDeveloper/z549ysk1/

## Information about environment
Nodejs or browser?
browser
Which browsers?
google chrome
## Steps to reproduce
The following errors appear on the console when we touch objects with two fingers
-----------------------------------
fabric.js:15052 Uncaught TypeError: this._setObjectScale is not a function
    at klass._scaleObjectBy (fabric.js:15052)
    at klass.__gesturesRenderer (fabric.js:14973)
    at klass.__onTransformGesture (fabric.js:14952)
    at klass._onGesture (fabric.js:13528)
    at HTMLCanvasElement.conf.onPointerDown (fabric.js:1538)
-----------------------------------
## Expected Behavior
It should scale when we touch objects resize with two fingers

**Comment by asturur** [comment]
Hi @ShahramCodeDeveloper, is this a build with gestures?

**Comment by ShahramCodeDeveloper** [comment]
hi asturur

yes

**Comment by TheMangoTrain** [comment]
Hey, still an issue in 4.0.0 rc1.

**WORKS: 3.6.1**
Pinch to scale/rotate object works
https://codepen.io/MarsAndBack/pen/PoZLWdP

**FAILS: 4.0.0 rc1** 
Pinch to scale/rotate object works
https://codepen.io/MarsAndBack/pen/zYrbNJL

Chrome console reports:
```
Uncaught TypeError: this._setObjectScale is not a function
    at i._scaleObjectBy (fabric.min.js:1)
    at i.__gesturesRenderer (fabric.min.js:1)
    at i.__onTransformGesture (fabric.min.js:1)
    at i._onGesture (fabric.min.js:1)
    at HTMLDocument.m.onPointerMove (fabric.min.js:1)
```

**Comment by asturur** [comment]
fixed with #6491
